soepy/solve/solve_auxiliary.py

```diff
@@ -95,6 +107,7 @@ def pyth_create_state_space(model_spec):
         model_spec.num_periods,
         model_spec.num_types,
         kids_ages.shape[0],
+        IND,
```

Comment:
If it is set up as a global config variable, then let's not also pass it through the interfaces.